src/test/scala/chiselTests/WireSpec.scala

```diff
@@ -29,7 +29,8 @@ class WireSpec extends ChiselFlatSpec {
       out := in & wire & wire2
     }
 
-    val chirrtl = ChiselStage.emitChirrtl(new Dummy)
+    val chirrtl = ChiselStage.emitCHIRRTL(new Dummy)
+    // This test can fail when run from IntelliJ because the source locator path can be different
     chirrtl should include("wire wire : UInt<1> @[src/test/scala/chiselTests/WireSpec.scala")
     chirrtl should include("wire wire2 : UInt<1> @[src/test/scala/chiselTests/WireSpec.scala")
```

Comment:
Can the source locator be removed instead?

Reply:
will do in next commit